app/models/cloud_tenant.rb

```diff
@@ -8,7 +8,6 @@ class CloudTenant < ApplicationRecord
   has_many   :cloud_subnets
   has_many   :network_ports
   has_many   :network_routers
-  has_many   :shared_cloud_networks, -> { where :shared => true }, :through => :ext_management_system, :source => :cloud_networks
```

Comment:
ah, good catch about the backwards compatibility case on automate :/

so keep it here, deal with the special case in `provision_workflow.rb`  and remove the alias in NetworkManager and everywhere else. So no need for a better naming then :)